src/Engines/TestWare.Engines.Selenium/Factory/InternetExplorerDriver.cs

```diff
@@ -0,0 +1,40 @@
+﻿
+
+using OpenQA.Selenium.IE;
+
+namespace TestWare.Engines.Selenium.Factory;
+
+public class InternetExplorerDriver : OpenQA.Selenium.IE.InternetExplorerDriver, IBrowserDriver
+{
+    public InternetExplorerDriver()

```

Comment:
Default constructor doesn't need to be specified
https://www.geeksforgeeks.org/c-sharp-default-constructor/

Reply:
deleted